### DISCUSSION TITLE
slide.addTable

### DISCUSSION THREAD

**Discussion by Strawberry0215** [gitbrent/PptxGenJS#1097] [General]
Hi @gitbrent I would like to ask if it is possible to addTable using Table id and not by rows?

for example; 
let rows = [["Cell 1", "Cell 2", "Cell 3"]];
slide.addTable(rows, { w: 9 });

// TABLE 2: Multi-row table
// - each row's array element is an array of cells
let rows = [["A1", "B1", "C1"]];
slide.addTable(rows, { align: "left", fontFace: "Arial" });

instead of defining the cell one by one, I want to use ID Table. Thank you

**Comment by gitbrent** [comment]
Are you looking for `tableToSlides()`?

### [HTML-to-PowerPoint Docs](https://gitbrent.github.io/PptxGenJS/html2pptx/)
